build/BUILD

```diff
@@ -96,6 +96,13 @@ genrule(
     cmd = "echo 0.5.1 >$@",
 )
 
+genrule(
+    name = "cri_tools_version",
```

Comment:
`cri_tools_package_version` and `cri_tools_version` to be consistent with cni above?

Reply:
yeah, that sounds right. best to be consistent here.